### PR TITLE
docs(hybrid) add status API docs for hybrid data plane

### PR DESCRIPTION
**Pull request by fffonion**
https://github.com/Kong/kong/pull/7429

### Review

@reviewer

### CODE REVIEW

**Review by kikito — APPROVED** [review]

**Comment by fffonion** [comment]
I think this feature is expanded in Kong Enterprise, where those endpoints can be prefixed with `/<workspace>`.
Where should I put the node mentioning those endpoints are workspace-aware? I know we also have a hybrid mode
doc for EE in like `app/enterprise/2.4.x/deployment/hybrid-mode.md`

File: app/gateway-oss/2.5.x/hybrid-mode.md [hunk 1]
```diff
@@ -505,6 +505,19 @@ means it has the same limitations as declarative config as of today. Please refe
 to the [Plugin Compatibility section][plugin-compat]
 of declarative config documentation for more information.
 
+## Readonly Status API endpoints on Data Plane
+
+Several readonly endpoints from [Admin API](/{{page.kong_version}}/admin-api)
```
Comment (HCloward) [inline app/gateway-oss/2.5.x/hybrid-mode.md:510]:
```suggestion
Several readonly endpoints from the [Admin API](/{{page.kong_version}}/admin-api)
```

File: app/gateway-oss/2.5.x/hybrid-mode.md [hunk 2]
```diff
@@ -505,6 +505,19 @@ means it has the same limitations as declarative config as of today. Please refe
...
+are exposed to [Status API](/{{page.kong_version}}/configuration/#status_listen) on Data Plane.
```
Comment (HCloward) [inline app/gateway-oss/2.5.x/hybrid-mode.md:511]:
```suggestion
are exposed to the [Status API](/{{page.kong_version}}/configuration/#status_listen) on data planes, including the following:
```

File: app/gateway-oss/2.5.x/hybrid-mode.md [hunk 3]
```diff
@@ -505,6 +505,19 @@ means it has the same limitations as declarative config as of today. Please refe
...
+
+- GET /upstreams/{upstream}/targets/
+- GET /upstreams/{upstream}/health/
+- GET /upstreams/{upstream}/targets/all/
+- GET /upstreams/{upstream}/targets/{target}
+
+Please refer to [Upstream objects](/{{page.kong_version}}/admin-api/#upstream-object) for above
```
Comment (HCloward) [inline app/gateway-oss/2.5.x/hybrid-mode.md:518]:
```suggestion
Please refer to [Upstream objects](/{{page.kong_version}}/admin-api/#upstream-object) for more information about the
```

**Review by HCloward — APPROVED** [review]
Minor wording edits - otherwise - LGTM - thanks @fffonion !

**Comment by HCloward** [comment]
> I think this feature is expanded in Kong Enterprise, where those endpoints can be prefixed with `/<workspace>`.
> Where should I put the note mentioning those endpoints are workspace-aware? I know we also have a hybrid mode
> doc for EE in like `app/enterprise/2.4.x/deployment/hybrid-mode.md`

So Enterprise has a Hybrid Mode Overview in the Deployment Options section of Installation here: https://docs.konghq.com/enterprise/2.4.x/deployment/hybrid-mode/ - I looked at the EE Admin API guide and did not see an endpoint that prefixes those endpoints with /workspace - so I am unsure what you are referencing.

File: app/gateway-oss/2.5.x/hybrid-mode.md [hunk 4]
```diff
@@ -505,6 +505,19 @@ means it has the same limitations as declarative config as of today. Please refe
 to the [Plugin Compatibility section][plugin-compat]
 of declarative config documentation for more information.
 
+## Readonly Status API endpoints on Data Plane
+
+Several readonly endpoints from [Admin API](/{{page.kong_version}}/admin-api)
+are exposed to [Status API](/{{page.kong_version}}/configuration/#status_listen) on Data Plane.
+
+- GET /upstreams/{upstream}/targets/
```
Comment (HCloward) [inline app/gateway-oss/2.5.x/hybrid-mode.md:513]: Should we link to the section of the Admin API where these live?

File: app/gateway-oss/2.5.x/hybrid-mode.md [hunk 5]
```diff
@@ -505,6 +505,19 @@ means it has the same limitations as declarative config as of today. Please refe
...
+- GET /upstreams/{upstream}/health/
```
Comment (HCloward) [inline app/gateway-oss/2.5.x/hybrid-mode.md:514]:
```suggestion
- [GET /upstreams/{upstream}/health/](/{{page.kong_version}}/admin-api/#show-upstream-health-for-node)
```

File: app/gateway-oss/2.5.x/hybrid-mode.md [hunk 6]
```diff
@@ -505,6 +505,19 @@ means it has the same limitations as declarative config as of today. Please refe
...
+- GET /upstreams/{upstream}/targets/all/
```
Comment (HCloward) [inline app/gateway-oss/2.5.x/hybrid-mode.md:515]:
```suggestion
- [GET /upstreams/{upstream}/targets/all/](/{{page.kong_version}}/admin-api/#list-all-targets)
```

**Pull request merged**